### DISCUSSION TITLE
Se puede cambiar de nombre data base (property) del web admin por colecciones u otro nombre

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#341] [2. Questions and Answers] [answered]
Hola

Es una pregunta más que un feature (dependiendo de la respuesta que obtenga creo o no una discusión). Resulta que me he dado cuenta que a veces confunde los dos data base que existe en cotalker en el momento de explicárselo a un cliente, como tambien internamente para los nuevos.

Si bien ambas tienen relación a "**properties**", pero con distintos focos.
![image](https://user-images.githubusercontent.com/61419471/126198595-d277b765-0f27-42d2-be53-9f7d71688c5b.png)

**Datos e Información**: Data Base: Permite ver colecciones, seleccionando un elemento y luego ver la información de ese elemento, para luego actualizar dicha información.

**Administración** / Administrador / Data Base: Permite ver colecciones, agregar una nueva colección, y crear/desactivar elementos. Además permite agregar los schemasnodes (extras 3.0) para la colección y que se refleje en los elementos para Data Base (Datos e Información).

**Qué nombres se usa en la configuración de formularios**

La sección corresponde a **Propiedades del Canal**, y te permite agregar **Elementos**, proveniente de una **Colección**.

![image](https://user-images.githubusercontent.com/61419471/126199148-a78c2d1b-60a0-4185-a18a-cdf05960afee.png)

Entonces creo que podría ser bueno, cambiar el nombre y colocarlo a Colección en vez de Base de Datos (para el admin), porque  se podría diferenciar entre los nombres existentes.

Gracias =>

**Comment by acedward** [comment, marked as answer]
Es una buena pregunta. Este es un tema dificil de nombrar! 
Te voy contar como está pensado ahora, y no es una respuesta a esta discusión.

Respecto a sus nombres, existen 3 niveles:

1. **Bases de Datos**: (Datos Maestros) Conjunto de Colecciones 
2. **Colección**: Conjunto de Elementos Homogéneos
3. **Elemento**: Fila individual

NOTA: Tanto el nombre propiedad y tipo-de-propiedad solo debe aparecer a nivel de API y nunca nunca en el Administrador.

Respecto al Administrador y la sección llamada "Base de Datos"
Son para usarios **diferentes**.

*  El Administrador es sólo para el administrador, quien configura la compañia, por eso tiene acceso a agregar schema-nodes.
*  La sección "Base de Datos" es para los usuarios, quienes NO deben entender la complejidad de la configuración.

**Reply by NicolasPLG** [reply]
hola @acedward muchas gracias por la información y sé que es complejo el tema. Entonces se debería primero reemplazar los nombres de propiedades que se encuentren en el administrador, para el nombre existente (colección) y luego ver más arriba?.

Gracias =>

**Reply by guillevalin** [reply]
Creo que el problema principal al menos de lo que expones es que en los formularios se hace referencia a "propiedades" cuando en realidad son "elementos" (que pertenecen a una colección), habría que revisar si existen otros casos de inconsistencias de este estilo, pero es algo que debemos considerar en un futuro para ir unificando nuestro lenguaje.

**Reply by acedward** [reply]
Mirando las traducciones, encontré al menos 4 instancias donde usamos la palabra "propiedad" deberíamos partir por ahi corrigiendo.

**Reply by acedward** [reply]
> Mirando las traducciones, encontré al menos 4 instancias donde usamos la palabra "propiedad" deberíamos partir por ahi corrigiendo.

Hemos manando una corrección para algunos lugares donde aun decía propiedades
